server/pbench/bin/index-pbench

```diff
@@ -1613,13 +1613,14 @@ def mk_metadata(fname, tb, mdconf, md5sum):
     """Return a dict with metadata about a tarball"""
     mtime = datetime.utcfromtimestamp(os.stat(fname)[stat.ST_MTIME])
 
-    mddict = {'generated-by': _NAME_,
-              'generated-by-version': _VERSION_,
-              'pbench-agent-version': mdconf.get("pbench", "rpm-version"),
-              'file-date': mtime.strftime("%Y-%m-%d"),
-              'file-name': fname,
-              'md5': md5sum
-    }
+    mddict = _dict_const([
+        ('generated-by', _NAME_),
+        ('generated-by-version', _VERSION_),
+        ('pbench-agent-version', mdconf.get("pbench", "rpm-version")),
+        ('file-date', mtime.strftime("%Y-%m-%d")),
+        ('file-name', fname),
+        ('md5', md5sum)
+    ])
```

Comment:
See previous comment. Are the dashes causing a problem here?

Reply:
Yes, a "-" in an identifier is not allowed, right?